src/game/client/components/broadcast.cpp

```diff
@@ -315,6 +315,16 @@ void CBroadcast::OnMessage(int MsgType, void* pRawMsg)
 				}
 			}
 
+			if(*c == CharUtf8 && *c == '\n')
```

Comment:
`*c == CharUtf8` is unnecessary, the `'\n'` is always a literal `\n` in UTF-8:

From [UTF-8 on Wikipedia](https://en.wikipedia.org/w/index.php?title=UTF-8&oldid=873448747):
>The first 128 characters of Unicode, which correspond one-to-one with **ASCII, are encoded using a single octet with the same binary value as ASCII**, so that valid ASCII text is valid UTF-8-encoded Unicode as well. Since **ASCII bytes do not occur when encoding non-ASCII code points into UTF-8**, UTF-8 is safe to use within most programming and document languages that interpret certain ASCII characters in a special way, such as "/" in filenames, "\" in escape sequences, and "%" in printf.

Reply:
Alright thanks @heinrich5991.